src/googledrive.js

```diff
@@ -144,31 +181,46 @@ GoogleDrive.prototype = {
         this._emit('connected');
         writeSettingsToCache.apply(this);
       } else {
-        this.info().then(function(info) {
+        this.info().then((info) => {
           this.userAddress = info.user.emailAddress;
           this._emit('connected');
           writeSettingsToCache.apply(this);
-        }.bind(this)).catch(function() {
+        }).catch(() => {
           handleError.apply(this);
           this.rs._emit('error', new Error('Could not fetch user info.'));
-        }.bind(this));
+        });
       }
     } else {
       handleError.apply(this);
     }
   },
 
+  /**
+   * Initiate the authorization flow's OAuth dance.
+   */
   connect: function () {
     this.rs.setBackend('googledrive');
     Authorize(this.rs, AUTH_URL, AUTH_SCOPE, String(Authorize.getLocation()), this.clientId);
   },
 
+  /**
+   * Stop the authorization process.
+   *
+   * @protected
+   */
   stopWaitingForToken: function () {
     if (!this.connected) {
       this._emit('not-connected');
     }
   },
 
+  /**
+   * Request a resource (file or directory).
+   *
+   * @param {string} path - Path of the resource
+   * @param {object} options - Request options
+   * @returns {Promise} Resolves with an object containing the status code, body, content-type and revision
```

Comment:
This is also private I think. A developer should only use `BaseClient` methods for get/put/delete calls.